index.js

```diff
@@ -224,14 +224,72 @@ switch (platform) {
         }
         break
       case 'arm':
+        if (isMusl()) {
```

Comment:
not sure why these changes are happening, do you run `yarn update` or something similar?

Reply:
@ozgrakkurt do you not get the same if you run `yarn build` or `yarn build:debug`?